bc-player.html

```diff
@@ -38,29 +38,71 @@ <h3 class="text-muted">Phil's Players</h3>
           <label for="manifest">MPEG DASH Manifest:</label>
           <input type="text" class="form-control" id="manifest" name="manifest">
         </div>
+        <div class="form-group">
+          <label for="license">Widevine License Server: <small>(Optional)</small></label>
+          <input type="text" class="form-control" id="license" name="license">
+        </div>
         <button type="submit" class="btn btn-primary btn-lg">Play!</button><br><br>
         <button type="button" id="demo" class="btn btn-info btn-sm">Load Demo Manifest</button>
       </form>
     </div>
 
     <div class="jumbotron">
-      <video id="example" width="600" height="300" data-account="4276901753001" data-player="edae5cc2-925e-4679-9984-89f75c7ddc31" data-embed="default" class="video-js vjs-default-skin center-block" controls></video>
-      <script src="//players.brightcove.net/4276901753001/edae5cc2-925e-4679-9984-89f75c7ddc31_default/index.min.js"></script>
+      <video id="example" data-account="5270290590001" data-player="BJszhI6rx" data-embed="default" data-application-id class="video-js vjs-default-skin center-block" controls width="600" height="300"></video>
+      <script src="//players.brightcove.net/5270290590001/BJszhI6rx_default/index.min.js"></script>
+
+      <!-- BC Player Debugger styles and script -->
+      <link href="https://solutions.brightcove.com/marguin/debugger/css/brightcove-player-debugger.css" rel="stylesheet">
+      <script src="https://solutions.brightcove.com/marguin/debugger/js/brightcove-player-debugger.min.js"></script>
     </div>
 
     <div class="jumbotron">
       <p>This player is The Brightcove Player with all DRM enabled.<br><a href="https://www.brightcove.com/en/online-video-platform/video-player">More details can be found here.</a></p>
       <p><small>This should play CENC content everywhere, using the Brightcove special sauce for setting license URLs, <a href="http://docs.brightcove.com/en/video-cloud/brightcove-player/guides/dash-drm.html">as documented here.</a></small></p>
```

Comment:
Should we note that this documentation isn't correct for license URLs, or just remove it now?